README.md

```diff
@@ -35,6 +177,75 @@ View the status of the installed agent
 vctl status
 ```
 
+To communicate with devices, one or more driver interfaces will also need to be installed.
+Each interface is distributed as a library and may be installed separately using pip.
+In the current RC version of the driver, only two interfaces are fully supported:
+
+* A Fake Driver (which returns data from a csv file):
+    ```shell
+    pip install volttron-lib-fake-driver
```

Comment:
We don't do this any longer...should be poetry add --directory $VOLTTRON_HOME volttron-lib-fake-driver until we get a wrapper command.